### ISSUE TITLE
[Content style guide ]The content is written in plain language and doesn't contain excess words or modifiers. (05.05.1)

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25183]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to be able to hear the end of the transcript message, then the input to ask another question immediately after. There's a lot of information before the text input that was confusing and didn't feel actionable.

## Details

There's several pieces of information being read out after I listen to the "End of chat" message before I can ask a new question when I'm using a screen reader. Specifically the following 4 pieces of information are read aloud:

* Connectivity Status: Connected
* Suggested Actions Container: Is empty, application status
* Suggested Actions Container: Is empty
* End of, Suggested Actions Container: Is empty, application status

Ideally these would be removed or hidden from screen readers unless they are important for the user experience.

## Acceptance Criteria

- [ ] Details of the ticket are considered by the development team and text is hidden from assistive technology OR
- [ ] Messages are updated to be meaningful or give users more context about what they are supposed to do

## Environment

* https://staging.va.gov/virtual-agent/

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

## Screenshots or Trace Logs

![Screen Shot 2021-05-24 at 4 39 54 PM](https://user-images.githubusercontent.com/934879/119410673-d5aabd00-bcae-11eb-85c0-f82f0e2e9619.png)

**Comment by ianpsant** [comment]
Requires Microsoft change. Would require assistance in filing Microsoft ticket.

**Comment by ianpsant** [comment]
Filed Microsoft bug mentioned above.

**Comment by noahgelman** [comment]
This doesn't have a 1-to-1 success criteria since it's more of a quality of life suggestion, but the following criteria is close.
WCAG Standard: [Success Criterion 1.3.2 Meaningful Sequence](https://www.w3.org/TR/WCAG21/#meaningful-sequence)

**Comment by ianpsant** [comment]
Fixed on Microsoft ticket https://github.com/microsoft/BotFramework-WebChat/issues/3938
